src/Features/Core/Portable/Diagnostics/IDiagnosticAnalyzerService.cs

```diff
@@ -74,49 +79,10 @@ internal interface IDiagnosticAnalyzerService
         /// </summary>
         Task<IEnumerable<DiagnosticData>> GetDiagnosticsForSpanAsync(Document document, TextSpan range, string? diagnosticIdOpt = null, bool includeSuppressedDiagnostics = false, Func<string, IDisposable?>? addOperationScope = null, CancellationToken cancellationToken = default);
 
-        /// <summary>
-        /// Gets a list of <see cref="DiagnosticAnalyzer"/>s for the given <see cref="Project"/>
-        /// </summary>
-        ImmutableArray<DiagnosticAnalyzer> GetDiagnosticAnalyzers(Project project);
```

Comment:
Should confirm if TypeScript uses any of these APIs.

Reply:
I did. It does not :)